synfig-studio/src/gui/dialogs/canvasresize.cpp

```diff
@@ -232,14 +201,43 @@ void CanvasResize::on_height_changed()
 	refresh_content();
 }
 
+bool CanvasResize::on_canvas_label_motion_event(GdkEventMotion *event)
+{
+	// Handle focus-hover-like event for canvas_only widget here
```

Comment:
Do you intend to focus on hovering? Why?

Reply:
For Checkbutton not label

But this might be removed if we go with ice0 layout.

Reply:
Default checkbutton behavior would highlight the button when mouse hovers it (both label and button). Since I used eventbox to make the checkbutton look like this https://github.com/synfig/synfig/pull/1895#issuecomment-842305438 hovering over the label doesn't indicate the checkbutton is being activated.

Reply:
What about Gtk CSS :hover ?

Reply:
> What about Gtk CSS :hover ?

That's what GTK team recommended too. I asked how to achieve hover behavior in their discussion forum yesternight
![Screenshot_2021-05-17_22-19-04](https://user-images.githubusercontent.com/24214509/118612048-071d1780-b7db-11eb-85e8-97e752c2f818.png)

I'm experimenting with the new layout suggested by ice0
![Screenshot_2021-05-18_13-13-47](https://user-images.githubusercontent.com/24214509/118611889-e81e8580-b7da-11eb-8e7c-4ccd5a12a373.png)

We might not have to worry about overriding GTK theme effect with CSS if placing checkbutton on the top left is suitable enough

Reply:
Figured how to do that

https://stackoverflow.com/questions/67736066/trigger-checkbutton-hover-event-from-another-widget